.github/workflows/pages-preview-cleanup.yml

```
@@ -0,0 +1,39 @@
+name: Cleanup PR Preview
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)